src/main/java/com/wavesplatform/wavesj/AssetDetails.java

```diff
@@ -0,0 +1,51 @@
+package com.wavesplatform.wavesj;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class AssetDetails {
+    public final String assetId;
+    public final Long issueHeight;
+    public final String issuer;
+    public final String name;
+    public final String description;
+    public final Integer decimals;
```

Comment:
Please follow common practice for java and make class members as private and provide corresponding getters methods

Reply:
Done.

Reply:
Agree